spec/features/adoption_applications/new_spec.rb

```diff
@@ -36,7 +36,7 @@
     expect(page).to have_content("Your adoption application has been submitted!")
 
     expect(current_path).to eq("/favorites")
-    expect(page).to_not have_content("Bonnie")
+    # expect(page).to_not have_content("Bonnie") - Bonnie now shows up due to User Story 18
```

Comment:
decide either to delete this expectation or specify that this content _should_ now be on the page, just within a specific css class